### PR TITLE
juju: Add audit support to kubernetes-master charm

### PR DESCRIPTION
**Pull request by Cynerva**
**What this PR does / why we need it**:

This PR adds audit configuration support to the kubernetes-master charm. Specifically, it enables audit logging by default, and adds two new config options: `audit-policy` and `audit-webhook-config`.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 1]
```diff
@@ -1166,8 +1180,9 @@ def configure_apiserver(etcd_connection_string, leader_etcd_version):
     api_opts['logtostderr'] = 'true'
     api_opts['insecure-bind-address'] = '127.0.0.1'
     api_opts['insecure-port'] = '8080'
-    api_opts['storage-backend'] = leader_etcd_version
+    api_opts['storage-backend'] = getStorageBackend()
```
Comment (ktsakalozos) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: Thank you for this

**Comment by Cynerva** [comment]
> New changes are detected. LGTM label has been removed.

I fixed a small bug: the charm was setting `audit-log-maxbackups`, but the correct config is `audit-log-maxbackup`.

**Pull request merged**